priv/repo/seeds.exs

```diff
@@ -143,3 +160,36 @@ units =
   end)
 
 Repo.insert_all(Unit, units, on_conflict: :nothing)
+
+currency_rewards =
+  Enum.flat_map(Enum.with_index(levels_without_units, 0), fn {level, level_index} ->
+    IO.inspect("Iterating")
+    currency_reward =
+      %CurrencyReward{
+        amount: 10 * level_index,
+        currency_id: gold_currency.id,
+        inserted_at: NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second),
+        updated_at: NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second)
+      }
+
+    %{level_id: level.id, currency_reward: currency_reward}
+  end)
+
+IO.inspect(currency_rewards, label: "currency_rewards")
```

Comment:
is this a leftover print? or is this supposed to be here?

Reply:
There are still some prints in the code that I use for debugging, I'll remove them as soon as the PR is ready for review